lib/patches.js

```diff
@@ -21,33 +21,38 @@ module.exports = function(Sequelize) {
          * Patches underscoredIf location differing in v2
          */
         underscoredIf: {
-          '^2.0.0': Sequelize.Utils._.underscoredIf,
-          '>=3.0.0': Sequelize.Utils.underscoredIf
+          '^2.0.0': Sequelize.Utils._? Sequelize.Utils._.underscoredIf : null,
+          '>=3.0.0 || >=5.0.0-beta': Sequelize.Utils.underscoredIf
         },
+        // For some reason in v5 uppercaseFirst is missing ... assume pluralize will do the job instead
+        uppercaseFirst: {
+            '>=5.0.0-beta': Sequelize.Utils.pluralize,
```

Comment:
`.pluralize()` does something else. I think this needs an explicit function e.g.:

```js
function(str) {
  return str.slice(0, 1).toUpperCase() + str.slice(1);
}
```